spec/javascript/components/LocationTileSpec.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+
+import LocationTile from '../../../app/javascript/react/components/LocationTile';
+
+describe('LocationTile', () => {
+  let id, name, wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <BrowserRouter>
+        <LocationTile
+          id="1"
+          name="Chipotle"
+        />
+      </BrowserRouter>
+    );
+  });
+
+  it("Should render a component with the specific props for the location name", () => {
+    expect(wrapper.find(LocationTile)).toHaveProp('name')
```

Comment:
Here, you're just testing that <LocationTile> has id and name props, which you know to be true because you defined it above. Instead, test that this presentational component is actually displaying the information as intended 

Example in article: 

  it("should render an h1 element containing the text received via props", () => {
    expect(wrapper.find("h1").text()).toBe("I am an Elephant!")
  })